### PR TITLE
Documentation Standard Updates

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
This pull request was created automatically 🎉:

- [x] Ensure `.github/` files have been properly updated (templates are updated and auto releaser is included)
- [x] Confirm all appropriate end models are included in the `.quickstart/quickstart.yml` file as `public_models` (only for transform repos)
- Verify the README has made all necessary updates:
    - [x] Removal of emojis
    - [x] Replacing exclamation points with periods in sentences
    - [x] Adjusting all headers to ensure only one H1 heading
    - [x] Confirming a section header is included for Quickstart UI rendering (only for transform repos)
    - [x] Renaming `**model` to `**table**` (only for transform repos)
    - [x] Removal of Fivetran community and office hour resources (if necessary)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,9 +29,9 @@
 - Generates a comprehensive data dictionary of your source and modeled Ad Reporting data via the [dbt docs site](https://fivetran.github.io/dbt_ad_reporting/)
 
 <!--section="ad_reporting_transformation_model"-->
-Refer to the table below for a detailed view of final models materialized by default within this package. Additionally, check out our [Docs site](https://fivetran.github.io/dbt_ad_reporting/#!/overview) for more details about these models. 
+Refer to the table below for a detailed view of final models materialized by default within this package. Additionally, check out our [Docs site](https://fivetran.github.io/dbt_ad_reporting/#!/overview) for more details about these models.
```
Comment (fivetran-joemarkiewicz) [inline README.md:32]:
```suggestion
Refer to the table below for a detailed view of final tables materialized by default within this package. Additionally, check out our [Docs site](https://fivetran.github.io/dbt_ad_reporting/#!/overview) for more details about these models.
```

Comment (fivetran-joemarkiewicz) [inline README.md:32]:
```suggestion
Refer to the table below for a detailed view of final tables materialized by default within this package. Additionally, check out our [Docs site](https://fivetran.github.io/dbt_ad_reporting/#!/overview) for more details about these models.
```

File: README.md [hunk 2]
```diff
@@ -29,9 +29,9 @@
...
 
-| **model**                  | **description**                                                                                                                                               |
+| **table**                  | **description**                                                                                                                                               |
```
Comment (fivetran-joemarkiewicz) [inline README.md:34]:
```suggestion
| **Table**                  | **Description**                                                                                                                                               |
```

File: README.md [hunk 3]
```diff
@@ -29,9 +29,9 @@
 - Generates a comprehensive data dictionary of your source and modeled Ad Reporting data via the [dbt docs site](https://fivetran.github.io/dbt_ad_reporting/)
 
 <!--section="ad_reporting_transformation_model"-->
-Refer to the table below for a detailed view of final models materialized by default within this package. Additionally, check out our [Docs site](https://fivetran.github.io/dbt_ad_reporting/#!/overview) for more details about these models. 
+Refer to the table below for a detailed view of final tables materialized by default within this package. Additionally, check out our [Docs site](https://fivetran.github.io/dbt_ad_reporting/#!/overview) for more details about these models.
```
Comment (fivetran-avinash) [inline README.md:32]: I see that model is renamed to table in this section but in no other sections, is this the only section in which these updates are needed?

File: .quickstart/quickstart.yml
Comment (fivetran-avinash) [inline .quickstart/quickstart.yml:1]: @fivetran-joemarkiewicz It looks like this PR is also including Quickstart configuration? Is there anything I need to review here before merging?

**Review by fivetran-avinash — COMMENTED** [review]
@fivetran-joemarkiewicz All the standard changes look good! Just a few comments before merge.

File: README.md (hunk 3 above)
Comment (fivetran-joemarkiewicz) [reply README.md:32]: Yes this is the only section since this is what is displayed in the UI

File: .quickstart/quickstart.yml
Comment (fivetran-joemarkiewicz) [reply .quickstart/quickstart.yml:1]: Nothing to review there, I already reviewed that as a separate PR.

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 4]
```diff
@@ -61,6 +67,19 @@ body:
         - other (mention it in "Additional Context")
     validations:
       required: true
+  - type: dropdown
+    id: orchestration_type
+    attributes:
+      label: How are you running this dbt package?
+      multiple: true
+      options:
+        - Quickstart
```
[the remark this answers is not in the document]
Comment (fivetran-avinash) [reply .github/ISSUE_TEMPLATE/bug-report.yml]: Updated.

**Review by fivetran-avinash — APPROVED** [review]
After some schema bumps to fix Buildkite errors, approved!

**Pull request merged**